src/mumble/LookConfig.cpp

```diff
@@ -297,7 +297,7 @@ void LookConfig::themeDirectoryChanged() {
 	qWarning() << "Theme directory changed";
 	QVariant themeData = qcbTheme->itemData(qcbTheme->currentIndex());
 	if (themeData.isNull()) {
-		reloadThemes(boost::none);
+        reloadThemes({});
```

Comment:
Maybe use `nullopt_t` for clarity? If so, we should also do it in the serialization test case generating Python script